concert_service_manager/src/concert_service_manager/utils.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
```

Comment:
We can extract out some function to `rocon_python_tools`
e.g)
*  setup_home_dirs -> create_cache_home
* get_home -> get_rocon_home